web_responsive/static/src/less/main.less

```diff
@@ -45,6 +45,15 @@ main {
     z-index: 1051;
 }
 
+
+// Remove z-index to preserve menu
+.o_control_panel {
+    .btn {
+        z-index: initial!important;
```

Comment:
Please don't use important, as it overwrites everything else. Use a more specific selector instead.